changelogs/.plugin-cache.yaml

```diff
@@ -31,6 +31,21 @@ plugins:
       name: email
       namespace: ''
       version_added: 1.5.0
+    file_dns:
+      description: Manage File DNS for PowerStore
+      name: file_dns
+      namespace: ''
+      version_added: 3.1.0
```

Comment:
I guess these changed need to update to their respective correct versions.

Reply:
this is a plugin cache file do you see it incorrect in changelog.yml as well ?, if can point out the correct version i can update,.
FYI, they are unchanged from previous release.

Reply:
this is a plugin cache file, please do you see it in changelog.yml/changelor.rst as well ?, if you can point out the correct version i can update and comment in changelog.yml i can update,.
FYI, they are unchanged from previous release.

Reply:
Sure,
I'll share the correct data.